sdl/units.go

```diff
@@ -51,9 +51,6 @@ func (u *cpuQuantity) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		if err != nil {
 			return err
 		}
-		if val < 0 {
-			return errNegativeValue
```

Comment:
`errNegativeValue` can probably be removed.

Reply:
It's still used below when parsing a float, I think it's fine to leave for now.